ansible_collections/arista/avd/docs/release-notes/3.x.x.md

```diff
@@ -16,105 +16,108 @@ No changes are required for existing deployments.
 
 #### Changes to eos_designs role
 
-- __Node Types variables:__
+- **Node Types variables:**
 
   - `node_type_keys.<key>.custom_templates_extra_vars` introduced in 3.7.0 has been removed. This is non-breaking since the templating environment
   used for custom templates now supports all host and group vars by default.
 
 ### Changes to requirements
 
-- AVD now requires ansible-core from __2.11.3__ to __2.12.x__ excluding __2.12.0__ to __2.12.5__<br>
-  Excluded versions have an [issue](## "plugin loader will now load config data for plugin by name instead of by file to avoid issues with the same file being loaded under different names (Fully-Qualified-Collection-Name + short-name).")
-  in `ansible-core`, which is [fixed](https://github.com/ansible/ansible/blob/v2.12.6/changelogs/CHANGELOG-v2.12.rst#bugfixes)
-  in __2.12.6__ and __2.13.1__.<br>
+- AVD now requires ansible-core from **2.11.3** to **2.12.x** excluding **2.12.0** to **2.12.5**<br>
+  Excluded versions have an issue in `ansible-core`, which is [fixed](https://github.com/ansible/ansible/blob/v2.12.6/changelogs/CHANGELOG-v2.12.rst#bugfixes)
+  in **2.12.6** and **2.13.1**.<br>
   If Ansible is installed using pip, a supported version can be installed with `pip3 install "ansible-core>=2.11.3,<2.13.0,!=2.12.0,!=2.12.1,!=2.12.2,!=2.12.3,!=2.12.4,!=2.12.5"`.
```

Comment:
Not sure we should still be saying `pip3` ... python 2 is supposed to be dead

Reply:
It is but sadly something or someone out there is using python2, pip3 is a nice way of saying "you better be using python3 :). We could add a minor PR with updates to all the pip3's found in the documentation.

Reply:
agreed!

Reply:
resolving this as it can be postponed to a later date